openslides/motions/config_variables.py

```diff
@@ -161,7 +161,7 @@ def get_config_variables():
 
     yield ConfigVariable(
         name='motions_comments',
-        default_value=[{'name': 'Comment', 'public': True}],
```

Comment:
Hm. I think one public comment field by default would be very nice for newcomers, right?